src/dune_rules/odoc.ml

```diff
@@ -404,7 +415,7 @@ let setup_toplevel_index_rule sctx =
 <html xmlns="http://www.w3.org/1999/xhtml">
   <head>
     <title>index</title>
-    <link rel="stylesheet" href="./odoc.css"/>
+    <link rel="stylesheet" href="./%s/odoc.css"/>
```

Comment:
One thing I wonder: this means that the html file has a path like `../../../odoc.css`, so that file can not be moved independently from the support files. Are those html files used for something or are they always consumed from within `_build`? In particular I'm wondering if there are cases where they are installed.

Reply:
That's true of all docs produced by odoc using a path as the 'support uri'; they calculate the path to the css/js from the uri supplied at html generation time. This doesn't change that aspect.

As far as I understand, the html files are never currently installed, so there's no extra problem there either.